### DISCUSSION TITLE
Expose port for elastalert

### DISCUSSION THREAD

**Discussion by karthiknagraj** [jertel/elastalert2#691] [Ideas]
Hi,
Currently, for the docker image available to use, Port is not exposed (please correct me if I am wrong). For my use case, I deploy elastalert into the Kubernetes cluster and I have a Microservice (built on operator SDK) that will pick alert rules from other applications running on my Kubernetes cluster and uploads those alert rules to elastalert. For me to achieve this, I need a port listening on elastalert. Can you please help me in exposing a port?
There are other advantages of opening a port. It can be used to perform liveness and readiness probe checks.

**Comment by nsano-rururu** [comment]
look
https://github.com/jertel/elastalert2/discussions/603

**Reply by karthiknagraj** [reply]
Are there any plans to enhance to provide a listening port?

**Reply by nsano-rururu** [reply]
I have no plans.

**Reply by karthiknagraj** [reply]
Thank You
